vllm/distributed/kv_transfer/kv_connector/v1/base.py

```diff
@@ -227,6 +227,16 @@ def get_finished(
         """
         return None, None
 
+    def get_block_ids_with_load_errors(self) -> set[int]:
+        """
+        Get the set of block IDs that failed to load.
+
+        Returns:
+            Set of block IDs that encountered load errors.
+            Empty set if no load errors occurred.
```

Comment:
I think we should be a bit more explicit about the semantics especially for the async case. i.e. failed blocks can be reported for async-loading requests in the steps up to and including the step where the request's id is returned from `get_finished()`; failed async loading requests must still report finished irrespective of failures.